flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/SystemOutRedirectionUtilsTest.java

```diff
@@ -77,26 +76,42 @@ void testDefaultSystemOutAndErr() {
         assertThat(errStream.toString()).isEqualTo(logContext);
     }
 
-    @ParameterizedTest
-    @EnumSource(
-            value = SystemOutMode.class,
-            names = {"IGNORE", "LOG"})
-    void testSystemOutAndErrAreRedirected(SystemOutMode systemOutMode) {
+    @Test
+    void testSystemOutAndErrAreRedirectedToLog() {
+        ByteArrayOutputStream outStream = new ByteArrayOutputStream();
+        ByteArrayOutputStream errStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outStream));
+        System.setErr(new PrintStream(errStream));
+
+        Configuration conf = new Configuration();
+        conf.set(TASK_MANAGER_SYSTEM_OUT_MODE, SystemOutMode.LOG);
+        SystemOutRedirectionUtils.redirectSystemOutAndError(conf);
+
+        String logContext = "This is log context!";
+        System.out.print(logContext);
+        assertThat(outStream.toString()).isEqualTo(SystemOutRedirectionUtils.OUT_TO_LOG_TIPS);
+
+        System.err.print(logContext);
+        assertThat(errStream.toString()).isEqualTo(SystemOutRedirectionUtils.ERR_TO_LOG_TIPS);
+    }
+
+    @Test
+    void testSystemOutAndErrAreRedirectedToIgnore() {
```

Comment:
We don't need 2 test methods here. The `SystemOutMode`, out tips and err tips could be argument.

You could refer to `DynamicFilteringITCase#testReuseDimSide` test[1], it uses 2 arguments. And `parameters` method [2]as the argument provider.

[1] https://github.com/apache/flink/blob/77e7ac076c35d229fab89b8b6ae15138b0e41a22/flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/DynamicFilteringITCase.java#L245
[2] https://github.com/apache/flink/blob/77e7ac076c35d229fab89b8b6ae15138b0e41a22/flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/DynamicFilteringITCase.java#L49

Reply:
@1996fanrui Thanks for the guidance, I made some changes, PTAL